samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
               * R - Data rate (Mbps)
               * P - PHY overhead duration (us) (Values: 24 us - Legacy OFDM, 192 us - DSSS)
 
+
```

Comment:
Please include an example of using RX mode, and also a follow up showing use of get_stats.

Reply:
Done